microsoft-edge/devtools-guide-chromium/whats-new/2022/09/devtools-105.md

```diff
@@ -0,0 +1,180 @@
+---
+title: What's new in DevTools (Microsoft Edge 105)
```

Comment:
delete file pngtemplate.png (nice to do)

Reply:
Done